src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectApproval.cs

```diff
@@ -41,6 +44,7 @@ public partial class StreetNameController
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
         [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.Adres.InterneBijwerker)]
```

Comment:
I think we need to register both policies (Adres.DecentraleBijwerker en Adres.InterneBijwerker) as separate attributes.